scripts/.Settings.js

```diff
@@ -1,5 +1,5 @@
 export const settings = {
-	graphhopperKey: "",
-	npsKey: "",
-	weatherKey: ""
+	graphhopperKey: "b9444e6b-2fec-424b-8c2e-4ae00f87c3f3",
+	npsKey: "zflqCsoWMQi4mVagiujDzfWb1TZZxTAAbkviJ1Hk",
```

Comment:
The NPS API key should be iS64Pl03HwTtlj5MKPdCon0gBhpKACS3fsxtq0kv

Reply:
changes made. I will re-submit my request.